src/config/nfs.md

```diff
@@ -0,0 +1,43 @@
+# NFS
+
+## Mounting an NFS Share
+
+To mount an NFS share, start by installing the `nfs-utils` package:
+
+```
+# xbps-install nfs-utils
+```
+
+Before you mount an NFS share, you'll need to [enable the statd and procbind
+services](services/index.md#enabling-services):
+
+(If your server supports `nfs4`, you do not need statd and you can replace `nfs`
+with `nfs4` below)
+
+```
+# sudo ln -s /etc/sv/rpcbind /var/service
+# sudo ln -s /etc/sv/statd /var/service
+```
+
+Now you're ready to mount a directory:
+
+```
+# mount -t nfs <host:/path/to/dir> /mnt/nfs/share
+```
+
+For example, if you wanted to connect `/volume` on a server at IP Address
+`192.168.1.99` to a directory `/mnt/volume` on your local system, you could do:
+
+```
+# mkdir /mnt/volume
+# mount -t nfs 192.168.1.99:/volume /mnt/volume
+```
```

Comment:
```
Before mounting an NFS share, it is necessary to [enable](./services/index.md#enabling-services) the `statd`
and `procbind` services. If the server supports `nfs4`, the `statd` service isn't necessary, and `nfs4` should be
used in the place of `nfs` as the mount type. To mount the NFS share, use the command below, where `host` is the
hostname or IP address of the server. Mounting options can be found in [mount.nfs(8)](https://man.voidlinux.org/mount.nfs.8), while unmounting ones can be found in [umount.nfs(8)](https://man.voidlinux.org/umount.nfs.8).

\```
# mount -t nfs host:/path/to/sourcedir /path/to/destdir
\```

```

Use relative paths for other files in the handbook. I'm not 100% happy with how the command is explained, though. You can ignore the backslashes, I just used them so the codeblock wouldn't break.